src/App.vue

```diff
@@ -14,13 +14,23 @@ export default {
     iiif_manifest_url() {
       const fixed_url = new URL(window.location.toString().replace("#?", "?")); // for some reason the URL format we inherited used '#?' to indicate query parameters, but URLSearchParams won't parse this.
       let iiif_url = fixed_url.searchParams.get("manifest");
+      // console.log(iiif_url);
+
+      // Need to account for test-iiif urls as well
       if (iiif_url.startsWith("https://iiif.library.ucla.edu/")) {
```

Comment:
Refactor with this if it works.
if (iiif_url.includes("library.ucla.edu/")) {
    let split_url[] = iiif_url.split("library.ucla.edu/");
    let ark = split_url[1].replace("/manifest", "");
    return `$split_url[0].library.ucla.edu/${encodeURIComponent(
              ark
            )}/manifest`;
 } else {
    return iiif_url;
}